tools/vm-migrator/Readme.md

```diff
@@ -0,0 +1,94 @@
+# GCP VM Migrator
+
+This utility automates migrating Virtual Machine instances within GCP. You can migrate VM's from one zone to another zone/region within the same project. 
+
+## Prerequisite
+
+To be able to execute the utility, the following prerequisites are to be met:
+1. Access to Code repository
+3. Python3 environment
+3. The service account with which you are running this script should be able to
+    * Create/Delete/Shutdown GCE
+    * Create/Delete VPC subnets
+    * Create/Delete machine images
+4. Ablity to create files on the filesytem to store the inventory data
+
+## Setup
+
+1. Clone the code repository
+2. cd in the code directoy
+3. run ```make install-requirements```
+4. run ```make install-tools```
+
+## Configuration
+The `Makefile` has different variables which are passed to the subnet_region_migratory.py these variables can be overriden from the command line 
+
+```make VARIABLE=VALUE migrate-subnet```
+
+
+| Variable      | Value                              |
+| ------------- | ---------------------------------- |
+| `PROJECT`          | GCP Project Name                  |
```

Comment:
In other places, it is also worth marking constants as code.  I just corrected one place as an example.
https://developers.google.com/style/code-in-text